src/App.tsx

```diff
@@ -1,14 +1,69 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
+const defaultTodo = {
+  id: 0,
+  title: 'no title',
+  completed: false,
+  userId: 0,
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoader, setIsLoader] = useState(true);
+  const [selectedTodo, setSelectedTodo] = useState<Todo>(defaultTodo);
```

Comment:
```suggestion
export const App: React.FC = () => {
  const [todos, setTodos] = useState<Todo[]>([]);
  const [isLoader, setIsLoader] = useState(true);
  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
```

Reply:
do you really need to create default todo? or you can just put null as a default value?)

Reply:
I can't do that because my selectedTodo and todo might return null and It'll be a mistake. However as far as I'm concerned I consider this to be a better practice than using null

Reply:
> I'm concerned I consider this to be a better practice than using null

I do not agree with this, if you have a shelf and you took everything from it, will you have a default item there? Or will you have nothing there?

Reply:
It is the same here, if you do not have a selected task, then you have nothing there and definitely not some default object